plugins/experimental/url_sig/url_sig.c

```diff
@@ -561,7 +561,9 @@ TSRemapDoRemap(void *ih, TSHttpTxn txnp, TSRemapRequestInfo *rri)
 
 /* ********* Allow ********* */
 allow:
-  app_qry = getAppQueryString(query, strlen(query));
+  if (query != NULL) {
+    app_qry = getAppQueryString(query, strlen(query));
+  }
```

Comment:
So this is the case where the exclusion regex was matched so it is ok to not have a query string?